identifiers/country-cy/top_level.csv

```diff
@@ -1,2 +1,8 @@
 id,name
 ocd-division/country:cy,Cyprus
+ocd-division/country:cy/district:01,Lefkosia

```

Comment:
I would use **ed** instead of **district**. WDYT ?

Reply:
I was going to do that at first, but it's really districts (https://en.wikipedia.org/wiki/Districts_of_Cyprus) and not electoral districts like in ecuador (https://es.wikipedia.org/wiki/Distritos_electorales_del_Ecuador).

But if everyone else thinks ed would be better anyway, i don't have a problem with that either? does anyone else have an opinion on this?

Reply:
I think district is fine/correct in this case

Reply:
Thanks for the clarification. Makes sense.

Reply:
`district` sounds good to me as well in this use-case